### ISSUE TITLE
[Enhancement]: Enable option to show m4b encoding progress/logs

### ISSUE THREAD

**Issue by TheArturthur** [advplyr/audiobookshelf#3141]
### Type of Enhancement

Web Interface/Frontend

### Describe the Feature/Enhancement

When encoding books with multiple files into one m4b file, there's no option to see the progress the ffmpeg process has.
This could be shown when selecting a "show logs/progress" option in the "use advanced option" section before starting the job.
Also, a time estimate could be given, updating with the task progress.

### Why would this be helpful?

The user could check how the encoding progress is going, how much time is left, and if there are some errors when encoding, that the ffmpeg logs would show.

### Future Implementation (Screenshot)

A possible view could be by showing a per-track progress, like when embedding metadata into the files:

Another one could be to simply show the logs, when selecting its option in the "use advanced options" section before encoding

### Audiobookshelf Server Version

2.11.0

**Comment by nichwall** [comment]
Duplicate of https://github.com/advplyr/audiobookshelf/issues/1986 but uses new template

**Comment by TheArturthur** [comment]
Sorry, didn't find that one, will close this one
